### PR TITLE
[rllib] Allow development without needing to compile Ray

### PR DESCRIPTION
**Pull request by ericl**
This adds a script that sets up symlinks between the rllib git dir and the installed source. Not perfect, but makes it so you don't need to compile Ray if just editing rllib source.

Closes https://github.com/ray-project/ray/issues/3608

### CODE REVIEW

File: python/ray/rllib/setup-rllib-dev.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    input = getattr(__builtin__, 'raw_input')
+except (ImportError, AttributeError):
+    pass
+
+if __name__ == "__main__":
+    rllib_home = os.path.abspath(os.path.join(ray.__file__, "../rllib"))
+    local_home = os.path.abspath(os.path.dirname(__file__))
+    assert os.path.isdir(rllib_home), rllib_home
+    assert os.path.isdir(local_home), local_home
+    print(
```
Comment (richardliaw) [inline python/ray/rllib/setup-rllib-dev.py]: `click.confirm(msg, abort=True)`?

**Review by richardliaw — APPROVED** [review]
LGTM and tried out on py2, py3 for macos. If you want to revert the last change, go ahead.

**Comment by ericl** [comment]
Lgtm

File: doc/source/installation.rst [hunk 2]
```diff
@@ -17,10 +17,6 @@ Trying snapshots from master
 
 Here are links to the latest wheels (which are built off of master). To install these wheels, run the following command:
 
-.. danger::
-
-    These versions will have newer features but are subject to more bugs. If you encounter crashes or other instabilities, please revert to the latest stable version.
```
Comment (robertnishihara) [inline doc/source/installation.rst]: Why remove this warning?

**Comment by robertnishihara** [comment]
So the idea is that you would `pip install ray` but then use `rllib` from the cloned repo?

**Comment by robertnishihara** [comment]
Ok sounds good. I'm fine with removing the warning.

**Pull request merged**